src/components/_proxied-dot-io/nomad/placement-table/index.jsx

```diff
@@ -0,0 +1,42 @@
+import { Fragment } from 'react'
+
+export default function PlacementTable({ groups = [] }) {
```

Comment:
I think this exists at `src/components/author-primitives/shared/placement-table/index.jsx` already 💭